deployments/fuji/usdc/deploy.ts

```diff
@@ -1,111 +1,109 @@
-      trace(await wait(WAVAX.connect(signer).deposit({ value: exp(0.01, 18) })));
-      trace(`WAVAX.balanceOf(${signer.address}): ${await WAVAX.balanceOf(signer.address)}`);
-    }
+  // Deploy Local Timelock
+  const localTimelock = await deploymentManager.deploy(
+    'timelock',
+    'vendor/Timelock.sol',
+    [
+      bridgeReceiver.address, // admin
+      10 * 60,                // delay
+      14 * SECONDS_PER_DAY,   // grace period
+      10 * 60,                // minimum delay
+      30 * SECONDS_PER_DAY    // maximum delay
+    ],
+    true // force
   );
 
-  // If we haven't spidered new contracts (which we could before minting, but its slow),
-  //  then the proxy contract won't have the impl functions yet, so just do it explicitly
-  const usdcProxy = contracts.get('USDC'), usdcImpl = contracts.get('USDC:implementation');
-  const USDC = usdcImpl.attach(usdcProxy.address);
+  // Initialize OptimismBridgeReceiver
   await deploymentManager.idempotent(
-    async () => (await USDC.balanceOf(fauceteer.address)).eq(0),
+    async () => !(await bridgeReceiver.initialized()),
     async () => {
-      trace(`Minting 100M USDC to fauceteer`);
-      const amount = exp(100_000_000, await USDC.decimals());
-      trace(await wait(USDC.connect(signer).configureMinter(signer.address, amount)));
-      trace(await wait(USDC.connect(signer).mint(fauceteer.address, amount)));
-      trace(`USDC.balanceOf(${fauceteer.address}): ${await USDC.balanceOf(fauceteer.address)}`);
+      trace(`Initializing BridgeReceiver`);
+      await bridgeReceiver.initialize(
+        GOERLI_TIMELOCK,      // govTimelock
+        localTimelock.address // localTimelock
+      );
+      trace(`BridgeReceiver initialized`);
     }
   );
 
-  const WBTC = contracts.get('WBTC.e');
-  await deploymentManager.idempotent(
-    async () => (await WBTC.balanceOf(fauceteer.address)).eq(0),
-    async () => {
-      trace(`Minting 10000 WBTC to fauceteer`);
-      const amount = exp(10000, await WBTC.decimals());
-      const feeAddress = '0x0000000000000000000000000000000000000000';
-      const feeAmount = 0;
-      const originTxId = '0x0000000000000000000000000000000000000000000000000000000000000000';
-      trace(await wait(WBTC.connect(signer).mint(fauceteer.address, amount, feeAddress, feeAmount, originTxId)));
-      trace(`WBTC.balanceOf(${fauceteer.address}): ${await WBTC.balanceOf(fauceteer.address)}`);
-    }
+  // Deploy Comet
+  const deployed = await deployComet(deploymentManager, deploySpec);
+  const { comet } = deployed;
+
+  // Deploy Bulker
+  const bulker = await deploymentManager.deploy(
+    'bulker',
+    'bulkers/BaseBulker.sol',
+    [
+      await comet.governor(), // admin
+      WETH.address            // weth
+    ]
   );
-}
+
+  // Deploy fauceteer
+  const fauceteer = await deploymentManager.deploy('fauceteer', 'test/Fauceteer.sol', []);
+
+  return {
+    ...deployed,
+    bridgeReceiver,
+    l2TelepathyRouter,
```

Comment:
I think you want to call this `telepathyRouter` right? Right now, this gets written into roots as `l2TelepathyRouter`.